docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.3'
+services:
+  bluefin-link:
+    build: .
+    image: bluefin-link
+    environment:
+      - AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+      - AWS_DEFAULT_OUTPUT=$AWS_DEFAULT_OUTPUT
+      - AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION
+      - AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      - CODECOV_TOKEN=$CODECOV_TOKEN
+      - NPM_AUTH_TOKEN=$NPM_AUTH_TOKEN
+      - DATABASE_URL=postgres://postgres:posgres@pg:5432/test
+    depends_on:
+      - pg
+
+  pg:
+    image: postgres:10-alpine
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_USER=postgres
```

Comment:
So I think we can leave the other connection strings in the test as `postgres:///test` if we just add these parameters to the environment:
```
- PGHOST=pg
- PGPORT=5432
- PGUSER=postgres
- PGPASS=postgres
```